token-lending/program/src/processor.rs

```diff
@@ -1584,8 +1584,13 @@ fn process_flash_loan(program_id: &Pubkey, amount: u64, accounts: &[AccountInfo]
         token_program: token_program_id.clone(),
     })?;
 
-    let (origination_fee, host_fee) = reserve.config.fees.calculate_flash_loan_fee(Decimal::from(amount), Exclusive)?;
-    let returned_amount_required = amount.checked_add(origination_fee).ok_or(LendingError::MathOverflow);
+    let (origination_fee, host_fee) = reserve
+        .config
+        .fees
+        .calculate_flash_loan_fee(Decimal::from(amount), Exclusive)?;
```

Comment:
If flash loan fees are always calculated exclusive of the amount, this argument should be removed and the behavior inlined to the implementation of `calculate_flash_loan_fees` (which should be plural)

Reply:
Good point. And I think it's plural already in the latest commit.